akka-stream-tck/src/test/scala/akka/stream/tck/GroupByTest.scala

```diff
@@ -17,7 +17,12 @@ class GroupByTest extends AkkaPublisherVerification[Int] {
     if (elements == 0) EmptyPublisher[Int]
     else {
       val futureGroupSource =
-        Source(iterable(elements)).groupBy(elem ⇒ "all").map { case (_, group) ⇒ group }.runWith(Sink.head)
+        Source(iterable(elements))
+          .groupBy(elem ⇒ "all")
+          .prefixAndTail(0)
```

Comment:
That is precisely the reason for:

* not documenting this purpose
* making it involve some boilerplate
* and in general being non-obvious

:-)